app/templates/components/signatures/timeline-step.hbs

```diff
@@ -35,12 +35,43 @@
             <!-- dit verschilt voor verschillende organen -->
             {{#wu-button
                class="button u-spacer--top--tiny"
-               onClick=(action sign mockDocument.signedId)
+               onClick=(action (mut disabledOnderteken) false)
                disabled=(not currentSession.canSign)
             }}
               Onderteken {{name}}
             {{/wu-button}}
           </div>
+          {{#unless disabledOnderteken}}
+            {{#wu-modal
+               title=(concat "Uw " currentRoute " wordt ondertekend")
+               dialog-class="modal-dialog--wide modal-dialog--sectioned"
+               onClose=(action (mut disabledOnderteken) true)
+               as |m|}}
+              <div class="modal-dialog__header">
```

Comment:
No need to add this `<div class="modal-dialog__header">` element. It's automatically created by the `wu-modal` component when you pass a `title` attribute